.env.default

```diff
@@ -8,8 +8,7 @@ SITE_NAME=Example
 SITE_MAIL=admin@example.com
 ADMIN_NAME=admin
 ADMIN_MAIL=admin@example.com
-ADMIN_PW=admin
-TESTER_PW=tester
+LOGIN_DESTINATION=admin/content
```

Comment:
Useless parameter. I dont see any reason to put it as variable

Reply:
Reason is simple: in some specific context, such as multi-site using sub dir mode, drupal doesn't support `/login?destination=`

So having it as variable will make it easy to disable that redirect (by leaving it var empty) if need be

Reply:
I would rather suggest to use `--no-browser` argument. so command may be
`....drush user:login --name="$(TESTER_NAME)" --no-browser" | awk....`